<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(24, 25, 25);
	border: none
}

/*----------------------------------------------------------------------------*/

QMenuBar {  
	background-color: rgb(24, 25, 25); /*rgb(17, 17, 18);*/
	spacing: 5px; /*spacing between menu bar items */
	border-bottom: 1px solid rgb(65, 65, 65);
	color:rgb(225, 225, 225);/*color font*/
}

QMenuBar::item {  
	padding: 2px 10px;
	background: transparent;  
	/*border-radius: 4px;*/
}

QMenuBar::item:selected, pressed { /* when selected using mouse or keyboard */  
	background-color: rgb(44, 43, 50);
}

/*----------------------------------------------------------------------------*/

QMenu {  
	
	color: rgb(199, 199, 199);
	background-color: rgb(44, 43, 50); /* sets background of the menu */  
	border:none;
}

QMenu::item {
	background-color: transparent;
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
	 background-color: rgb(120, 117, 130);
}

QMenu::separator { 
 	height: 1px; 
	background: rgb(78, 78, 79);  
	margin: 1px 5px;
}

/*----------------------------------------------------------------------------*/

QPushButton{
	color: white;
	background-color: rgb(44, 43, 50);
	border: none;
}

QPushButton::hover{
	background-color:rgb(43, 50, 76);
}

QPushButton::pressed{
	background-color:rgb(78, 78, 79);
}

/*----------------------------------------------------------------------------*/
QScrollBar {
     border: none;
     background: #494949;
     height: 6px;
     width: 6px;
     margin: 0px;
}

QScrollBar::handle {
     background: #DBDBDB;
     min-width: 20px;
     min-height: 20px;
}

 QScrollBar::add-line,  QScrollBar::sub-line {
     background: none;
     border: none;
}

 QScrollBar::add-page, QScrollBar::sub-page {
     background: none;
}

/*----------------------------------------------------------------------------*/
QTreeView {
	show-decoration-selected: 1;
 	background: rgb(71, 71, 71);
	alternate-background-color: rgb(82, 82, 82);
	border:0px;
	selection-background-color: none;
}
QTreeView::item {
	color: rgb(199, 199, 199);
}

QTreeView::item:focus {    
    background-color: rgb(61, 60, 71);
}

QTreeView::item:selected {    
    background-color: transparent;
}

QHeaderView::section {  
	background:rgb(87, 87, 85); 
 	color: rgb(199, 199, 199); 
	border: 0px;
	border-left: 2px solid rgb(99, 101, 98);
	padding: 1px 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>361</width>
      <height>501</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QChartView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>691</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="ChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_JSON"/>
    <addaction name="actionOpen_CVS"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_from_network"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_the_program"/>
   </widget>
   <widget class="QMenu" name="menuEditing">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_a_graph_from_the_internet"/>
    <addaction name="actionClear_the_charts"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEditing"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen_JSON">
   <property name="text">
    <string>Open JSON</string>
   </property>
  </action>
  <action name="actionOpen_CVS">
   <property name="text">
    <string>Open CSV</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_from_network">
   <property name="text">
    <string>Open from network</string>
   </property>
  </action>
  <action name="actionAbout_the_program">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAdd_a_graph_from_the_internet">
   <property name="text">
    <string>Add a graph internet</string>
   </property>
  </action>
  <action name="actionClear_the_charts">
   <property name="text">
    <string>Clear the charts</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
  <customwidget>
   <class>ChartView</class>
   <extends>QGraphicsView</extends>
   <header>chartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
